docs/maps/connect-to-ems.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[maps-connect-to-ems]]
+== Connecting to Elastic Maps Service
+
+https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
+tile layers and vector shapes of administrative boundaries.
+With Kibana's out-of-the-box settings, the **Maps** application uses EMS services.
```

Comment:
Does this work:

If you are using Kibana's out-of-the box settings, the **Maps** application is already configured to use EMS.